hack/jenkins/windows_integration_setup.ps1

```diff
@@ -12,9 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-$test_root="$env:HOMEDRIVE$env:HOMEPATH\minikube-integration"
-$test_home="$test_root\$env:COMMIT"
+$test_home="$env:HOMEDRIVE$env:HOMEPATH\minikube-integration"
```

Comment:
this is effectively changing what $test_home is actually set to. is that safe?

Reply:
I originally set it to a commit  specific folder because I thought that would prevent the described problem. However, I forgot to take into that these tests are run a schedule and the same commit on head could be run multiple times.

Also, if I used a commit specific folder and the teardown script doesn't run the folder will stay there forever. Using the same folder for every test will ensure it's always properly wiped.

Reply:
gotcha, makes sense